.woodpecker/main.yml

```diff
@@ -17,6 +17,8 @@ pipeline:
     image: golang:1.16
     commands:
       - make release-agent
+    when:
+      event: tag
```

Comment:
You added this as `publish-agent-dryrun` will now do the build test?

Reply:
Yes we only need this for the deb files etc ... and that one is only triggered on tag